tests/data/issues/1540

```diff
@@ -1,5 +1,23 @@
 {
   "number": 1540,
   "title": "A mock open issue for testing",
-  "state": "open"
+  "state": "open",
+  "labels": [
+      {
+        "name": "refactor",
+        "color": "47f441"
```

Comment:
Some of these should include a `url:` prop, as it's expected with `createLabels()`.

Reply:
Yep sure

Reply:
Now I guess there is no need of the `url:` prop, as we are generating our own urls

Reply:
Correct!